### DISCUSSION TITLE
Multiple users for one account

### DISCUSSION THREAD

**Discussion by ghost** [deso-protocol/dips#61] [Medium CIPs]
Gradually, corporate companies started to open Bitclout accounts. In these companies, the personnel responsible for social media manage the account. This situation creates a problem. If the employee is malicious, it can damage the account. It can steal all the funds in the account. I think there should be a multi-user feature. And the authority of these accounts should be restricted. (authority to manage the fund, send diamonds, send messages, edit bio)

**Comment by omicreativedev** [comment]
Cloutmegazord has already made this. It's more of a 3rd party dapp type of thing.

**Reply by ghost** [reply]
To be honest, I don't like 3rd party apps. They are not a value without Bitclout. At this stage, Bitclout's infrastructure needs to be strengthened. After the Bitclout infrastructure is strengthened, 3rd party applications should be supported. 

So I don't want to take up too much of your time. The feature I mentioned above is absolutely necessary. People have difficulty even opening an account on Bitclout.com. Not all people are at your level of consciousness. You can't tell these people about a 3rd party app. The less complex the platform is, the more people will prefer it.

**Comment by tijno** [comment]
The team is already looking at derived keys i believe that may solve this.
